Gemfile

```diff
@@ -1,12 +1,20 @@
 source 'https://rubygems.org'
 
-ruby '2.5.0'
+ruby '2.6.0'
 
+gem 'bcrypt'
+gem 'pg'
+gem 'rack'
 gem 'rake'
-gem 'rubocop', '0.56.0'
+gem 'shotgun'
+gem 'sinatra'
+gem 'sinatra-flash'
```

Comment:
good gem for notices!

Reply:
This is good for personalised 'flash notices' but then you should not use UNIQUE when constructing your columns in the database!